integration-tests/src/test/resources/domain-home-on-pv/create-domain-auto-custom-sit-config.py

```diff
@@ -116,6 +116,13 @@ def getEnvVar(var):
   cmo.setCluster(cl)
   print('Done setting attributes for Server Template: %s' % templateName);
 
+  templateChannelName = cluster_name + "-NAP"
```

Comment:
This file is used in testAutoAndCustomSitConfigOverrides, domain is not restarted. are you sure this config is needed here?

Reply:
I took a look at the code, you are right. I thought we fun the full lifecycle test here too, but we don't. I'll take it out.